Sources/JSONService.swift

```diff
@@ -0,0 +1,70 @@
+
+/// Default implementation of a JSON service.
+/// Serves your `Order` to a server and parses the respons.
+/// Response is delivered to you as a `JSONResult`.
+public class JSONService : Service {
+    private var task: NSURLSessionDataTask?
+
+    /// Always results in .Success(["key" : "value"])
+    /// This will change to a real request in the future
+    override public func serve <M : Mappable> (order: Order, result: (Result<M>) -> ()) {
+        guard let url = configuration.url else {
+            result(.Failure)
+            return
+        }
+
+        let fullUrl = url.URLByAppendingPathComponent(order.path)
+        let mutableRequest = NSMutableURLRequest(URL: url, cachePolicy: .ReloadIgnoringLocalAndRemoteCacheData, timeoutInterval: 10.0)
+         mutableRequest.HTTPMethod = order.method.rawValue
+
+        let session = NSURLSession.sharedSession()
+        task = session.dataTaskWithURL(fullUrl, completionHandler: { (data, response, error) in
+            if let data = self.checkStatusCodeAndData(data, urlResponse: response, error: error) {
+                do {
+                    let json = try NSJSONSerialization.JSONObjectWithData(data, options: .AllowFragments)
+                    result(.JSON(json))
+                }catch {
+                    result(.Failure)
+                }
+            }else {
+                result(.Failure)
+            }
+        })
+
+        task?.resume()
+    }
+
+    func cancel() {
+        task?.cancel()
+    }
+
+    private func checkStatusCodeAndData(data: NSData?, urlResponse: NSURLResponse?, error: NSError?) -> NSData? {
+
+        guard error == nil else {
```

Comment:
❓When the error is not nil that you have a status code above or equal to 400. So I don't think you have to manually check for the status codes?